composer.json

```diff
@@ -10,22 +12,20 @@
     "flysystem",
     "cloud logging"
   ],
-  "homepage": "http://github.com/leon0399/monolog-flysytem",
-  "type": "library",
-  "license": "MIT",
   "authors": [
     {
       "name": "Leonid Meleshin",
       "email": "leon.03.99@gmail.com"
     }
   ],
+  "homepage": "http://github.com/leon0399/monolog-flysytem",
   "require": {
-    "php": "^7.2",
-    "monolog/monolog": "^2.0",
-    "league/flysystem": "^1.0"
+    "php": ">=7.3",
```

Comment:
Wouldn't it be better to use `>=7.4 || ^8`?

Reply:
I'm pretty sure `>=7.4` also covers `^8` as well (in that `>=7.4` is equivalent to `^7.4 | ^8`). I just chose 7.3 as the minimum as it's the minimum that phpunit 9 supports. 

I can change it to which ever you prefer. I just personally need it compatible with the currently supported PHP version (8.1).